src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -159,7 +159,8 @@ int main(void)
     /* USER CODE BEGIN WHILE */
     while (1)
     {
-        // TODO: Confirm with akhil if this should still compile with freertos enabled
+        // TODO: Confirm with akhil if this should still compile with legacy_freertos
```

Comment:
TODO

Reply:
removed, this was from an old PR

Reply:
@garethellis0  Was this TODO supposed to be removed?

Reply:
@garethellis0 Was this TODO supposed to be removed?